server/services/caseloadService.ts

```diff
@@ -203,13 +203,13 @@ export default class CaseloadService {
       .filter(offender => !offender.nomisRecord.paroleEligibilityDate)
       .filter(offender => offender.nomisRecord.legalStatus !== 'DEAD')
       .filter(offender => !offender.nomisRecord.indeterminateSentence)
-      .filter(offender => offender.nomisRecord.conditionalReleaseDate)
-      // TODO: Following filter rule can be removed after 18th April 2022
-      .filter(offender =>
-        moment(offender.nomisRecord.conditionalReleaseDate, 'YYYY-MM-DD').isSameOrAfter(
-          moment('2022-04-18', 'YYYY-MM-DD'),
-          'day'
-        )
+      // TODO - Check with Jon on rules for filtering recalls
+      // .filter(offender => !offender.nomisRecord.recall)
```

Comment:
I think one of the cases was a `recall==true` - but really it was not on the case list because it had no CRD, and instead had a confirmedReleaseDate and releaseDate.

Reply:
Yea the recall flag was a red herring I think. AFAIK the only things that were causing problems was the lack of CRD and the HDC filter rules